contracts/CollateralTracker.sol

```diff
@@ -850,7 +850,7 @@ contract CollateralTracker is ERC20Minimal, Multicall {
         uint160 currentSqrtPriceX96 = TickMath.getSqrtRatioAtTick(currentTick);
         uint160 medianSqrtPriceX96 = TickMath.getSqrtRatioAtTick(medianTick);
         unchecked {
-            for (uint256 leg = 0; leg < TokenId.countLegs(positionId); ++leg) {
+            for (uint256 leg; leg < TokenId.countLegs(positionId); ++leg) {
```

Comment:
Please revert. Although this is logically identical, we explicitly specify =0 here for clarity.